docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
     env_file:
       - ./.docker/local.env
     environment:
-      MYSQL_ALLOW_EMPTY_PASSWORD: true
+      MYSQL_ALLOW_EMPTY_PASSWORD: "true"
```

Comment:
boolean are not allowed, changing to string

Reply:
@sri-kirsh sure, it automatically converts to a string, which maps to truthy. pretty common dc hack.

Reply:
(docker-compose syntax has no sense of a boolean)

Reply:
ya that's what i thought too - but it was throwing error for me in my local machine when it was set as boolean
```
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.mysql.environment.MYSQL_ALLOW_EMPTY_PASSWORD contains true, which is an invalid type, it should be a string, number, or a null
```

Reply:
weird, okay thanks for sharing your experience here.